main.js

```diff
@@ -77,7 +77,11 @@ const Expander = function (el, opts) {
 	this.configure('countSelector', '.' + this.contentClassName + ' > li');
 	this.configure('expandedToggleText', this.opts.shrinkTo === 'hidden' ? 'hide' : this.opts.shrinkTo === 'height' ? 'less' : 'fewer');
 	this.configure('collapsedToggleText', this.opts.shrinkTo === 'hidden' ? 'show' : 'more');
-	this.configure('toggleSelector', 'a.' + this.toggleClassName);
+	if (this.el.querySelector('a.' + this.toggleClassName)) {
```

Comment:
If I make that change a `button` list and a link list cannot live in the same document, should I leave it as it is, maybe?